pkg/volume/csi/csi_attacher.go

```diff
@@ -269,6 +269,25 @@ func (c *csiAttacher) MountDevice(spec *volume.Spec, devicePath string, deviceMo
 		return err
 	}
 
+	dataDir := filepath.Dir(deviceMountPath)
```

Comment:
Won't this end up putting the file in the same dir that is mounted?
On the mount side, it saves the file in `/blah/savefile.txt` and puts the mount in `/blah/mount/`

Reply:
No, that's why there is `filepath.Dir`. It ends up in `/var/lib/kubelet/plugins/kubernetes.io/csi/pv/pvc-938861e4756911e8/vol_data.json` and the volume is mounted into `/var/lib/kubelet/plugins/kubernetes.io/csi/pv/pvc-938861e4756911e8/globalmount`.